src/tests/unit/lottery-application.test.ts

```diff
@@ -0,0 +1,34 @@
+import {describe, expect} from '@jest/globals';
```

Comment:
Great addition. Tests are always important on any app. Even this small exercise. **Well done!**